tests/integration/test_ext_hms.py

```diff
@@ -103,7 +103,7 @@ def test_migration_job_ext_hms(ws, installation_ctx, prepare_tables_for_migratio
 
     tables, dst_schema = prepare_tables_for_migration
     ext_hms_ctx = installation_ctx.replace(
-        config_transform=lambda wc: dataclasses.replace(wc, override_clusters=None),
+        config_transform=lambda wc: dataclasses.replace(wc, override_clusters={"main": "key:table_migration"}),
```

Comment:
We can't fetch the table migration cluster id from the test instead of the prefix with key route?

Reply:
`table_migration` is a job cluster that needs to be created - this is necessary for this test because we set spark_conf for external hms explicitly